src/include/txn_inline.h

```diff
@@ -719,6 +719,21 @@ __txn_visible_all_id(WT_SESSION_IMPL *session, uint64_t id)
     return (WT_TXNID_LT(id, oldest_id));
 }
 
+/*
+ * __wt_txn_timestamp_visible_all --
+ *     Can the provided timestamp be globally visible?
+ */
+static WT_INLINE bool
+__wt_txn_timestamp_visible_all(WT_SESSION_IMPL *session, wt_timestamp_t timestamp)
+{
+    wt_timestamp_t pinned_ts;
+
+    /* If no oldest timestamp has been supplied, updates have to stay in cache. */
```

Comment:
What do you think of:

```suggestion
    /* Compare the given timestamp to the pinned timestamp, if it exists. */
```

Reply:
Fixed.